assets/css/style.css

```diff
@@ -5,7 +5,7 @@
      box-sizing: border-box;
 }
 html{
-     font-family: "Lato", sans-serif;
+    font-family: Verdana, sans-serif !important;
 }
```

Comment:
không nên để **important** chỗ này vì html tag dùng cho toàn bộ trang. mà 1 trang thì có thể có nhiều font family khác nhau.